gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -346,7 +347,7 @@ def __init__(self, file_path):
     self._file_path = file_path
     self._file_to_write = None
 
-  def process(self, header):
+  def process(self, header, *args, **kwargs):
```

Comment:
I don't think we should add args/kwargs. These are used for optional side-inputs (they get replaced with appropriate variables, see [here](https://beam.apache.org/documentation/programming-guide/#passing-side-inputs-to-pardo)) and none of the existing python implementations (even outside of this project) have them.

Reply:
I realized that we don't use them in our code but without these, we get a warning that this signature "does not match signature of base method" and that is because DoFn.process has them. So from a pure base class implementation perspective I thought they should be there (BTW, is there any harm in having them?). I do see that even in Beam documentation examples, they are not always present so I removed them.

Reply:
I prefer not having them as they are distracting and not a convention used anywhere else. You can consider disabling this warning at the cost of potentially masking other legit warnings (I'm ok with disabling if the warnings are too annoying).